### PR TITLE
LLM Observability updated SDK setup docs

### PR DESCRIPTION
**Pull request by lievan**
### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/tracing/llm_observability/sdk.md [hunk 1]
```diff
@@ -12,22 +12,46 @@ You can install and configure tracing of various operations such as workflows, t
...
+    dd_llmobs_no_apm=True,
+    integrations=[LLMObs.openai, LLMObs.botocore, LLMObs.langchain],
+)
+{{< /code-block >}}
 
-Your application startup command must specify the required environment variables:
+
+#### Command line setup
+
+You can also enable LLM Observability by running your application using the `ddtrace-run` command and specifying the required environment variables. Note that `ddtrace-run` automatically turns on all LLM Observability integrations.
```
Comment (rtrieu) [inline content/en/tracing/llm_observability/sdk.md:48]:
```suggestion
You can also enable LLM Observability by running your application using the `ddtrace-run` command and specifying the required environment variables.

**Note**: `ddtrace-run` automatically turns on all LLM Observability integrations.
```

File: content/en/tracing/llm_observability/sdk.md [hunk 2]
```diff
@@ -346,6 +354,11 @@ def process_message():
     return 
 {{< /code-block >}}
 
+
+## Force flushing in serverless environments
+
+To gaurantee spans are sent in serverless environments, use the `LLMObs.force_flush()` function right before your application exits. This function blocks until all LLM Observability data has been submitted to the Datadog backend.
```
Comment (rtrieu) [inline content/en/tracing/llm_observability/sdk.md:360]:
```suggestion
To guarantee spans are sent in serverless environments, use the `LLMObs.force_flush()` function right before your application exits. This function blocks until all LLM Observability data has been submitted to the Datadog backend.
```

Comment (rtrieu) [reply content/en/tracing/llm_observability/sdk.md:360]: Is a word between "blocks" and "until" missing? What does the function block? Or is it blocked?

**Review by rtrieu — CHANGES_REQUESTED** [review]
Left some minor feedback for your review!

File: content/en/tracing/llm_observability/sdk.md (hunk 2 above)
Comment (lievan) [reply content/en/tracing/llm_observability/sdk.md]: Just updated the description to clarify!

**Review by rtrieu — APPROVED** [review]
Looks good, thank you!

**Pull request merged**